### ISSUE TITLE
Android self managed mode example

### ISSUE THREAD

**Issue by Baer05** [react-native-webrtc/react-native-callkeep#583]
would it be possible for a detailed example on Android self managed mode to be added to the project? how exactly does a java class have to look like so that your own CallUi can be displayed?

**Comment by abhinav0031** [comment]
yes please a example would be best, as a person who has mainly only worked on react native, having a good example about what we need to implement in native side to make it work would be great.

**Comment by adha-fajri** [comment]
have you guys found a code example on this? If not then can anyone who's knowledgable on this help please?
